hw_diag/diagnostics/add_gateway_txn_diagnostic.py

```diff
@@ -0,0 +1,42 @@
+import json
+import secrets
+
+from hm_pyhelper.diagnostics import Diagnostic
+from hm_pyhelper.constants.shipping import DESTINATION_WALLETS_KEY
+from hm_pyhelper.miner_json_rpc.exceptions import MinerMalformedAddGatewayTxn
+
+from hw_diag.utilities.miner import create_add_gateway_txn
+from hw_diag.utilities.security import get_verified_data
+
+KEY = 'ADD_GATEWAY_TXN'
```

Comment:
KEY and FRIENDLY_NAME should be the same thing. They are only different to support legacy code that used an ugly format.